src/c++/devices/tracer/dmTracer.c

```diff
@@ -19,6 +19,18 @@
 #include "common.h"
 #include "traceLoggerBlockTrace.h"
 
+#ifndef VDO_UPSTREAM
+#undef VDO_USE_ALTERNATE
+#if defined(RHEL_RELEASE_CODE) && defined(RHEL_MINOR) && (RHEL_MINOR < 50)
+#if (RHEL_RELEASE_CODE < RHEL_RELEASE_VERSION(10, 0))
+#define VDO_USE_ALTERNATE
+#endif
+#else /* !RHEL_RELEASE_CODE */
+#if (LINUX_VERSION_CODE < KERNEL_VERSION(6, 12, 0))
+#define VDO_USE_ALTERNATE
+#endif
+#endif /* !RHEL_RELEASE_CODE */
+#endif /* !VDO_UPSTREAM */
```

Comment:
It only just occurred to me, but we normally put this block next to the code where VDO_USE_ALERNATE is used. This should move down to around line 330.

Reply:
I have removed the VDO_USE_ALTERNATE block completely